### DISCUSSION TITLE
Ability to disable set homes.

### DISCUSSION THREAD

**Discussion by FestusDM** [Zrips/CMI#2476] [Q&A]
**Question: **
I'm curious as if there's a way to disable set homes for players.
For example. They can set home in the overworld but not in the End or Nether.

Is it a feature that CMI has? 
Thanks for the help.

**Comment by quorn23** [comment]
Usually your permission plugin is responsible for the context your command works. You can do such things with Luckperms for example. hope that helps.

**Comment by FestusDM** [comment]
Right but as far as I know, Luckperms are used for server wide permissions. 
I just want to set it so that players can set home in the overworld but not in the End or Nether?

Unless you know a certain permission that sets that. It'd be helpful to know what command to run.

**Comment by quorn23** [comment]
[https://github.com/lucko/LuckPerms/wiki/Advanced-Setup](https://github.com/lucko/LuckPerms/wiki/Advanced-Setup)

I didn't fully read through it, but you set the permission as negative one (as false instead of true i assume) for the specific world.

Edit: quickly looking through the perm section on Zirps page i didn't find anything regarding defining the world within a permission

**Comment by FestusDM** [comment]
Thanks for the link, will have a read. 
It'd just be nice to have something inside of CMI; to keep everything under one roof sort of thing.

**Comment by CloudeLecaw** [comment]
disable the permissions in the certains worlds, 
done
would be nice if you close the ticket ;)

**Comment by CloudeLecaw** [comment]
so pls clode this discussion :-)
